src/line_absorption.jl

```diff
@@ -36,10 +36,10 @@ function line_absorption!(α, linelist, λs::Wavelengths, temps, nₑ, n_densiti
     β = @. 1 / (kboltz_eV * temps)
 
     # precompute number density / partition function for each species in the linelist
-    n_div_Z = map(unique([l.species for l in linelist])) do spec
+    n_div_U = map(unique([l.species for l in linelist])) do spec
```

Comment:
This change is purely for terminological consistency.  Partition functions are spelled "`U`" in the Korg codebase